extensions/GradioNotebook/gradionotebook/frontend/styles.css

```diff
@@ -0,0 +1,82 @@
+/* 
```

Comment:
Also self-reminder, we can probably get rid of most of this with updated GradioTheme as well